jsb.js

```diff
@@ -379,118 +368,26 @@ jsb = {
          * We need to use concat, because otherwise the returned array would
          * change as soon as we remove the element's className.
          */
-        var raw_dom_elements = null;
-        if (dom_element.getElementsByClassName) {
-            raw_dom_elements = dom_element.getElementsByClassName(this.prefix);
```

Comment:
We need this for IE8 compat:

http://caniuse.com/#feat=queryselector

Should be ok then.